node-data/src/encoding.rs

```diff
@@ -414,6 +415,25 @@ impl Serializable for ValidationResult {
     }
 }
 
+impl Serializable for ValidationQuorum {
```

Comment:
I didn't get why we need a new message (ValidationQuorum) instead of reusing the `Ratification vote` msg that already includes `ValidationResult`

```rust
 pub async fn try_vote(
        ru: &RoundUpdate,
        iteration: u8,
        result: &ValidationResult,
        outbound: AsyncQueue<Message>,
    ) -> Message {
        // Sign and construct ratification message
        let ratification =
            self::build_ratification_payload(ru, iteration, result);

        let msg = Message::from(ratification);
 ```

Reply:
Ratification message would fit the goal too, I agree

I just believe that it starts to become a mess when we have to extract ValidationResult from there (in term of code branch).
Saying this because I cannot envision a clear distinction between the choice to just aggregate a ratification vote (as for now) or the need to extract the ValidationResult to vote for

I'm open to any suggestion

Reply:
Anyway, I'm going to remove validation quorum from this pr completely

Reply:
Using the Ratification vote limits the transmission of ValidationResult to Ratification voters only, or those who received (and stored) a Ratification vote.
I'm much more inclined towards a separate message